### PR TITLE
[NETBEANS-1074] Module Review cloud.amazon

### PR DESCRIPTION
**Pull request by arusinha**
External zip cloud.amazon-external-resources.zip was deleted instead content of it "empty.war" was created at compile-time

### CODE REVIEW

File: cloud.amazon/licenseinfo.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/cloud/amazon/resources/empty.war</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/amazon.png</file>
+  	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/running.png</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/terminated.png</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/tomcat.png</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/waiting.png</file>
```
Comment (junichi11) [inline cloud.amazon/licenseinfo.xml]: Please fix indentation.

File: cloud.amazon/build.xml [hunk 2]
```diff
@@ -25,7 +25,6 @@
 
     <property name="resourcesdir" value="./src/org/netbeans/modules/cloud/amazon/resources/"/>
     <target name="build-init" depends="projectized.build-init">
-        <unzip src="external/cloud.amazon-external-resources.zip" dest="${resourcesdir}"/>
     </target>
```
Comment (junichi11) [inline cloud.amazon/build.xml]: Probably, we should remove the following? they are unused, right?
```
    <property name="resourcesdir" value="./src/org/netbeans/modules/cloud/amazon/resources/"/>
    <target name="build-init" depends="projectized.build-init">
    </target>
```

File: cloud.amazon/licenseinfo.xml (hunk 1 above)
Comment (arusinha) [reply cloud.amazon/licenseinfo.xml]: Has corrected now the indentation. In view still it is not showing properly.
Plz check the original file
https://github.com/apache/incubator-netbeans/blob/732e41fe8185aa4286116fff4e4483d9eb207bf8/cloud.amazon/licenseinfo.xml

File: cloud.amazon/build.xml (hunk 2 above)
Comment (arusinha) [reply cloud.amazon/build.xml]: Had removed now those extra lines

**Comment by matthiasblaesing** [comment]
I wanted to merge, but then I noticed the new binary `empty.war`. That file was not donated by oracle. The file is trivial and could be created at runtime/compile time.  Both relevant files should be just redone.

**Comment by arusinha** [comment]
Had now  recreated web.xml and welcome.jsp file manually, Please verify the changes

**Comment by matthiasblaesing** [comment]
Looks good to me. Thank you.

**Pull request merged**